Example/TABTestKit/ViewController.swift

```diff
@@ -9,15 +9,57 @@
 //
 
 import UIKit
+import LocalAuthentication
 
 class ViewController: UIViewController {
-  @IBOutlet var helloWorldLabel: UILabel!
-  @IBOutlet var testTextField: UITextField!
+  @IBOutlet private var helloWorldLabel: UILabel!
+  @IBOutlet private var testTextField: UITextField!
+  @IBOutlet private var biometricsStateLabel: UILabel!
+  @IBOutlet private var authenticateButton: UIButton!
   
-    override func viewDidLoad() {
-      helloWorldLabel.text = "Hello world!"
-      testTextField.accessibilityIdentifier = "Test Text Field"
-      helloWorldLabel.accessibilityIdentifier = "Hello World Label"
+  
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    configure()
+    setupNotificationListener()
+    setup()
+  }
+  
+  private func configure() {
+    helloWorldLabel.text = "Hello world!"
+    testTextField.accessibilityIdentifier = "Test Text Field"
+    helloWorldLabel.accessibilityIdentifier = "Hello World Label"
+  }
+  
+  private func setupNotificationListener() {
+    NotificationCenter.default.addObserver(self, selector: #selector(setup), name: NSNotification.Name.UIApplicationDidBecomeActive, object: nil)
```

Comment:
Do we still need this?

Reply:
Yeah, because it will update the state of the label when I background and foreground the app. I can also remove the code from the UI test that checks the label text ("Biometrics unavailable" or "Biometrics available"). Perhaps I can also use KVO to update the label when the biometrics get enabled, but not sure if it's possible.

Reply:
Can we not just terminate the app and relaunch it?

Reply:
Yeah I guess we can do that, I’ll give it a shot soon!

Reply:
Awesome. Goes back to what I was saying before about trying to avoid adding code specifically for testing wherever possible!

Reply:
Yeah, I have removed it now! Doing a relaunch as well instead of backgrounding/foregrounding.